lesson_05/davidsmith/README.md

```diff
@@ -0,0 +1,7 @@
+## User Stories
+
+* As a new user I want my webpage react to the size of the window.
```

Comment:
Missing the last part of the formulation.

`As a [ROLE], I want to [ACTION], so that [PURPOSE]`